asulibraries/islandora-repo
# Newcontent digest 235 - ready to re-review and merge

Provided that the views can be imported into the VM this can be merged.

There seemed to be a problem with theming the email on the KEEP site and it may have to do with an attempt to rename the view display from "differences : page_2" to "differences : differences1day" - yet upon review on my local VM, a full config:import led to a themed email again.

To test
1. configure SMTP to send email to at least one account that you have an email account attached
2. without editing or adding any content in the last 24 hrs, navigate to the System Status page and click the [Run cron] button to see an example of the emailing when no content has been changed or added.
3. create a new node that is unpublished so it appears in the "New content" report because once the moderation state is changed from Draft to Published, that will make it display on the "Changed content" report
3. change at least one other node so that appears on the "Changed content" report.
4. navigate to the System Status page and click the [Run cron] button to see an example of the reports that display changed as well as new content in a single emailing

I hate to say that "fingers should be crossed" when the branch is tested and the email is sent.

Closes the issue: https://github.com/asulibraries/islandora-repo/issues/235
